File: Lista1/Java/ex_3.java
import java.util.Scanner;
public class ex_3
{
	public static void main(String[] args) {
	    Scanner sc = new Scanner(System.in);
	    int qtd_pares = 0;
	    for (int c = 1; c<=10; c++) {
	      System.out.println("Digite o " + c + "° numero: ");
	      int numero = sc.nextInt();
	      if (numero % 2 == 0) {
            qtd_pares ++;
          }
	    }
		System.out.println("O número de valores pares digitados é: " + qtd_pares);
	}
}

File: Lista1/Python/ex_3.py
qtd_pares = 0
for c in range(1,11):
    numero = int(input(f'Digite o {c}° número: '))
    if numero % 2 == 0:
        qtd_pares += 1
print(f'A quantidade de números pares digitados foi: {qtd_pares}')